src/sipTransportConstructor.ts

```diff
@@ -71,6 +75,10 @@ var computeRandomTimeout = (
     return randomInterval + retryOffset;
 };
 
+async function __connect (this: WebPhoneSIPTransport, options?: any): Promise<void>{
+   return await this.__connect(options).catch((err)=>{ this.logger.log('Error while connecting: '+ err)});
```

Comment:
We need to emit some event here, console log is not enough.

Reply:
I agree @kirill-konshin  , @Alex-us  based on the emitted event , we would want to force reconnect when the reconnection attempt fails, should it be directly handled via SDK or should the client handle this error?